src/block_implementations/Eos/EosEncoderBlock.cpp

```diff
@@ -44,8 +50,15 @@ void EosEncoderBlock::onMidiMessage(MidiEvent event) {
                 relativeValue -= 128;
             }
             // accelerate:
-            relativeValue = (qAbs(relativeValue) > 1) ? relativeValue * 2 : relativeValue;
-            QString message = m_fineMode ? "/eos/user/1/wheel/fine/%1" : "/eos/user/1/wheel/coarse/%1";
+            if (m_accelerate)
+                relativeValue = (qAbs(relativeValue) > 1) ? relativeValue * 2 : relativeValue;
+
+            QString message;
+            if (!m_active)
+                message = m_fineMode ? "/eos/user/1/wheel/fine/%1" : "/eos/user/1/wheel/coarse/%1";
+            else
+                message = m_fineMode ? "/eos/active/wheel/fine/%1" : "/eos/active/wheel/coarse/%1";
+
             message = message.arg(m_parameterName);
```

Comment:
Please add a hint to the helpText in the header file how to use the 'active' mode, i.e. that the parameter name is then the number of the active fader.

Reply:
described in header